app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardClustersStep/components/ClustersStepForm/ClustersStepForm.js

```diff
@@ -116,6 +118,12 @@ class ClustersStepForm extends React.Component {
       />
     );
 
+    const targetConversionHostWarning = __(`
+      At least one host in the target cluster must be enabled as a conversion host. You can
+      continue to create an infrastructure mapping that includes the target cluster, but
+      conversion host enablement must be completed prior to migration execution.
+    `);
```

Comment:
The i18n rake task has a limitation that it can't extract strings placed in the above multi-line format.

Hence what we could do is make sure that the entire text is wrapped up in one single line as shown below -
```js
__("Looooooooooooooooooooooooooong line") // prettier-ignore
```

Reply:
Aw jeez, I was hoping we wouldn't have to do that. I'll fix it, thanks @AparnaKarve